### PR TITLE
[PATCH Artwork] Fix Lightbox on mobile, restructure storybook entries

### PR DESCRIPTION
**Pull request by damassi**
- Fixes Lightbox (reattaches to image, and locks zoom when swiping on mobile)
- Reorganizes storybooks files into `Apps -> Components` 
- Light renaming for consistency

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f673903b0c87261967dac153b4c896d11c006df4

https://deploy-preview-1702--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 9c96502216472d33f68e8208d03b6454ff060b58

https://deploy-preview-1702--artsy-reaction.netlify.com

File: __mocks__/openseadragon.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react"
+
+export default function openseadragon() {
```
Comment (damassi) [inline __mocks__/openseadragon.js]: This is our lightbox file

File: src/Apps/Artwork/Components/ArtworkBrowser/ArtworkBrowser.tsx [hunk 2]
```diff
@@ -126,6 +91,78 @@ export class LargeImageCarousel extends React.Component<ImageBrowserProps> {
...
+    })
+  }
+
+  unlock = () => {
+    setTimeout(() => {
+      this.setState({
+        isLocked: false,
+      })
+    }, 10)
+  }
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkBrowser/ArtworkBrowser.tsx]: Will need to qa this a bit more on a real device but will check once we send out to staging.

File: src/Apps/Artwork/ArtworkApp.tsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ export const ArtworkApp: React.SFC<Props> = props => {
       <Row>
         <Col sm={8}>
           <Box px={[0, 4]}>
-            <ImageBrowser artwork={props.artwork} />
+            <ArtworkBrowser artwork={props.artwork} />
```
Comment (oxaudo) [inline src/Apps/Artwork/ArtworkApp.tsx]: How come we renamed that to ArtworkBrowser? Is't that a component that just does the slideshow and zoom on Images. ArtworkBrowser suggests that one will be flipping through artworks in my mind.

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx [hunk 4]
```diff
@@ -70,7 +70,7 @@ export const ArtworkContextAuctionFragmentContainer = createFragmentContainer<{
     const OtherAuctionsList = (
       <OtherAuctions>
         {sales.map(sale => {
-          return <AuctionCard sale={sale} />
+          return <AuctionCard sale={sale} key={(sale as any).id} />
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx]: How come `as any` needed for using in the key assignment but not for sale assignment?

File: src/Apps/Artwork/Components/__stories__/ArtworkBrowser/ArtworkBrowser.story.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { Box } from "@artsy/palette"
+import { ImageBrowserQueryRenderer } from "Apps/Artwork/Components/ArtworkBrowser"
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { Col } from "Styleguide/Elements/Grid"
+import { Section } from "Styleguide/Utils/Section"
+
+storiesOf("Apps/Artwork Page/Components/ArtworkBrowser", module).add(
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/__stories__/ArtworkBrowser/ArtworkBrowser.story.tsx]: And here we are keeping it as ImageBrowser?

**Pull request merged**